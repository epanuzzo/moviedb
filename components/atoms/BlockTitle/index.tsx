import React from "react";

type BlockTitleProps = {
  children: string;
};

const BlockTitle: React.FC<BlockTitleProps> = ({ children }) => {
  return <h3 className="text-2xl font-semibold mb-4">{children}</h3>;
};

export default BlockTitle;
