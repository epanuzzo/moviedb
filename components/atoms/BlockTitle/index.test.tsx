import React from "react";
import renderer from "react-test-renderer";
import BlockTitle from "./index";

describe("BlockTitle", () => {
  it("renders component", () => {
    const tree = renderer.create(<BlockTitle>Test</BlockTitle>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
