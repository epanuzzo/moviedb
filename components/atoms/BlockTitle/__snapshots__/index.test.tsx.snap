// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockTitle renders component 1`] = `
<h3
  className="text-2xl font-semibold mb-4"
>
  Test
</h3>
`;
